sites/svelte.dev/vite.config.js

```diff
@@ -2,8 +2,26 @@ import { sveltekit } from '@sveltejs/kit/vite';
 import { browserslistToTargets } from 'lightningcss';
 import { readFile } from 'node:fs/promises';
 import browserslist from 'browserslist';
+import { transformWithEsbuild } from 'vite';
 
-const plugins = [raw(['.ttf']), sveltekit()];
+/** @type {import('vite').Plugin[]} */
+const plugins = [
+	raw(['.ttf']),
+	sveltekit(),
+	{
+		name: 'minified-raw-js',
+
+		async load(id) {
+			if (id.endsWith('.js?minified')) {
```

Comment:
a more unique query string and resolveId hook with `\0` tagging would be more defensive here.

Reply:
I've been preferring `\0virtual:` lately to add some clarity